src/Engine/Rendering/UIRenderer.cpp

```
@@ -17,6 +17,10 @@ UIRenderer::UIRenderer(GLFWwindow *window, const Ctx::VkRenderDevice *renderDevi
     bindEvents();
     initImGui();
 
+    // Set baseline style ONCE (to revert back to on DPI changes)
```

Cppcheck (2.20.0)

[style] The function 'ToSimulationSpace' is never used.

(unusedFunction)